server/catgenome/src/main/java/com/epam/catgenome/controller/vcf/VcfController.java

```diff
@@ -75,18 +77,17 @@
  * It's designed to communicate with corresponded managers that provide all required
  * calls and manage all operations concerned with a VCF.
  */
-@Controller
+@RestController
+@RequiredArgsConstructor
 @Api(value = "VCF", description = "VCF Track Management")
 public class VcfController extends AbstractRESTController {
 
-    @Autowired
-    private VcfSecurityService vcfSecurityService;
+    private final VcfSecurityService vcfSecurityService;
 
     @Value("${vcf.load.info.for.track:false}")
```

Comment:
Could you please check that value injection will work for such mixed constructor + field injection?